apps/meeting/src/containers/MeetingControls/index.tsx

```diff
@@ -16,16 +17,17 @@ import {
 import EndMeetingControl from '../EndMeetingControl';
 import { useNavigation } from '../../providers/NavigationProvider';
 import { StyledControls } from './Styled';
+import { useAppState } from '../../providers/AppStateProvider';
 
-const MeetingControls = () => {
+const MeetingControls = (): JSX.Element => {
```

Comment:
Shall we use the `React.FC` instead of `JSX.Element`?

Reply:
done